README.md

```diff
@@ -101,47 +101,69 @@ Follow the steps below to set up a testnet smart wallet using `starkli`:
 
 ## Declare and Deploy Contracts in Testnet
 
+### Ethereum smart contract
+First, the Ethereum smart contracts must be deployed. For Ethereum the deployment process you will need to:
+
+1. Create your `.env` file: you need to configure the following variables in your own .env file on the contracts/solidity/ folder. You can use the env.example file as a template for creating your .env file, paying special attention to the formats provided
```

Comment:
> You can use the env.example file as a template for creating your .env file, paying special attention to the formats provided
Which format provided?

Reply:
The format provided is in the .env.example files